CHANGELOG.md

```diff
@@ -29,12 +33,15 @@
 * [ENHANCEMENT] Store-gateway: improved performance of series matching. #3391
 * [ENHANCEMENT] Move the validation of incoming series before the distributor's forwarding functionality, so that we don't forward invalid series. #3386 #3458
 * [ENHANCEMENT] S3 bucket configuration now validates that the endpoint does not have the bucket name prefix. #3414
+* [ENHANCEMENT] Store-gateway: index stats response. #3206
+* [ENHANCEMENT] Distributor: push wrapper should only receive unforwarded samples. #2980
 * [BUGFIX] Flusher: Add `Overrides` as a dependency to prevent panics when starting with `-target=flusher`. #3151
 * [BUGFIX] Updated `golang.org/x/text` dependency to fix CVE-2022-32149. #3285
 * [BUGFIX] Query-frontend: properly close gRPC streams to the query-scheduler to stop memory and goroutines leak. #3302
 * [BUGFIX] Ruler: persist evaluation delay configured in the rulegroup. #3392
 * [BUGFIX] Ring status pages: show 100% ownership as "100%", not "1e+02%". #3435
 * [BUGFIX] Fix panics in OTLP ingest path when parse errors exist. #3538
+* [BUGFIX] Canonicalize orgid header in forwarding. #3385
```

Comment:
I would just add this PR number next to 3283. This isn't a bugfix between releases since we started to include this header only in Mimir 2.5.0.

Reply:
Done: 45b55e81d